East Indies Reworked 3.5 - Radja Jawa

KrasnaMerah opened the pull request:
So far:
- Updating the Monarch possibilities
- Adding proper idea portraits for the monarchs
- Adding a few advisors
- Adding events for the historical capitulation and for the monarchy stuffs
- Updated portraits for advisors

Hopefully i can have time for:
- Some coherent forpol focus for independent Indonesia

KrasnaMerah committed:
Update 030225

Updating the Monarch possibilities

(i'm not done yet, Marijn, don't close this)

KrasnaMerah committed:
Update 020325 pt2

Adding some events and actually recruiting the characters

marijn211 commented:
Please keep as draft while not done

KrasnaMerah committed:
Update 040225

Proper events for the monarchs

KrasnaMerah committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Update 050225

Adding Nicolaas Blom and "switch to Van Mook" event, further pathing for the portraits

KrasnaMerah committed:
Update 060225

Hopefully the last one

Leader and advisor portraits, also event pictures

KrasnaMerah commented:
Okay, i'm done

KrasnaMerah commented:
Bruuuhhhh

marijn211 committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

marijn211 reviewed (changes requested):
[21:57:52][1936.01.01.12][effectimplementation.cpp:5392]: history/countries/INS - Indonesia.txt:105: recruit_character: Unknown character

KrasnaMerah committed:
Update 140225

Adding Mangkunegara VII to the mod

KrasnaMerah commented:
> [21:57:52][1936.01.01.12][effectimplementation.cpp:5392]: history/countries/INS - Indonesia.txt:105: recruit_character: Unknown character

Yeah, i forgot to add him, he already has a portrait there.

Thanks.
![image](https://github.com/user-attachments/assets/847217ca-d452-4113-8b03-8d84f70564cf)

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Update 160225

Adding Papuan flags based on the Koreri movement, also readding Moluccan core on Aru Islands, and removing Palangkaraya

KrasnaMerah committed:
Update 160225 pt2

Some more Papuan names and parties

marijn211 reviewed (approved):
Passed the eyeball test, passed the in-game test, passed the error log test and up-to-date with main

marijn211 commented:
Reverted to draft as Krasna said he wants to add some stuff

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Update 230225 - WP and Indo COM flags

Changes to West Papuan and Indonesian communist flags

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
010325 - Moar Indies Portraits

Updating more portraits
